src/main.ts

```diff
@@ -96,6 +99,7 @@ async function run(): Promise<void> {
     } else if (trivyStatus === 0) {
         console.log("No vulnerabilities were detected in the container image");
     } else {
+        utils.extractErrorsFromLogs(trivyHelper.getTrivyLogPath());
```

Comment:
Take the tool name as an input; just in case if the logic changes.

Reply:
The function is present in both trivyHelper and dockleHelper because it has different definitions for both the tools. I don't think tool name needs to be passed as an argument. If it's ever needed, it can directly be accessed (There's a const variable for tool name in both the helpers)

Reply:
Done